charts/postgres/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+type: application
+name: postgres
+description: Helm Chart for redisearch
+icon: https://www.asserts.ai/favicon.png
+version: 0.1.0
+appVersion: "12.7"
```

Comment:
Need to change this to a triple digit version (will need to match up with our custom image).